### DISCUSSION TITLE
Issue with alerts in console

### DISCUSSION THREAD

**Discussion by AdelNorberg** [vuejs/pinia#1070] [Help and Questions] [answered]
Can you please tell me how to remove this? :(
![image](https://user-images.githubusercontent.com/24225624/154425875-831f4409-7c17-4bfe-bdfd-6a7d703b1b4d.png)
I tried to find how to turn off these notifications, but I did not understand

**Comment by posva** [comment, marked as answer]
You can't yet. Added it to https://github.com/vuejs/pinia/issues/732
